src/leiningen/github_cdn.clj

```diff
@@ -63,23 +66,23 @@
           (println "Removing old files")
           (git "rm" "-rf" "*" {:dir *dir* :throw false})
           (copy-files src-dir *dir*)
-          (println "Commiting and pushing")
+          (println "Committing and pushing")
           (git "add" "*" {:dir *dir* :throw false})
           (git "commit" "-m" comment' {:dir *dir* :throw false})
           (git "push" "origin" (str branch ":" branch) {:dir *dir*})
           (println "Done")))
       (println "No source dir specified"))))
 
 (defn github-cdn
-  "Publishes resources to gihhub branch"
+  "Publishes resources to gh-pages branch"
```

Comment:
let's set it to `a github branch` since `gh-pages` is the default one and it can be configured via `:gihtub-cdn { :branch "value" }` the same about change below as it's a commit comment `Updated files on github pages` is a better name